external-crates/move/crates/move-vm-runtime/src/dev_utils/in_memory_test_adapter.rs

```diff
@@ -120,7 +120,8 @@ impl VMTestAdapter<InMemoryStorage> for InMemoryTestAdapter {
         package: SerializedPackage,
     ) -> VMResult<(verif_ast::Package, MoveVM<'extensions>)> {
         let Some(storage_id) = package.linkage_table.get(&runtime_id).cloned() else {
-            // TODO: VM error instead?
+            // NB: This is a panic because it means something went really wrong -- we should always
+            // panic if we expect a package to exist and it is not there.
```

Comment:
This is for test-only so this doesn't matter, but for the "real one" I think an InvariantViolation would be best as that will allow the network to keep progressing on other transactions.